scripts/offload-idea.js

```
@@ -0,0 +1,84 @@
+    type: 'string',
+    demandOption: true,
+  })
+  .option('source', {
+    alias: 's',
+    describe: 'Source of the idea',
+    type: 'string',
+  })
+  .help()
+  .argv;
+
+const type = argv.type;
```

Biome (1.9.4)

[error] Shouldn't redeclare 'type'. Consider to delete it or rename it.

'type' is defined here:


(lint/suspicious/noRedeclare)